### PR TITLE
fixes #1570 broken layout code

### LINKED ISSUE #1570: [bug] broken code layout with text fully wrapped
1. Go to https://webcompat.com/issues/2009#issuecomment-304553096
2. See how the code is wrapped.

capture d ecran 2017-05-29 a 10 16 25

Compare with github.
https://github.com/webcompat/web-bugs/issues/2009#issuecomment-304553096

### PR DESCRIPTION
**Pull request by magsout**
r? @karlcow 

I have created two comments in this issue http://localhost:5000/issues/667 for the review.

(if you local is sync with https://github.com/webcompat/webcompat-tests/issues/667 )

### CODE REVIEW

File: webcompat/static/css/development/components/markdown.css [hunk 1]
```diff
@@ -21,9 +21,8 @@
   /* all element */
   .wc-Markdown * {
     font-size: var(--Markdown-fontSize) !important;
-    word-wrap: break-word;
-    white-space: normal;
```
Comment (karlcow) [inline webcompat/static/css/development/components/markdown.css]: These were introduced to fix the code overflow in 
https://github.com/webcompat/webcompat.com/commit/1b6b80d92c30d14b616f9dfc5222b567f6185c43

Comment (karlcow) [inline webcompat/static/css/development/components/markdown.css]: I have the feeling there is a simpler fix. 

```css
.wc-Markdown code {
	white-space: pre-wrap;
}
```

Comment (magsout) [inline webcompat/static/css/development/components/markdown.css]: Altogether, it was not necessary to add this part. It was enough to target <code>, I wanted to clean up a bit the Markdown code. Maybe I should not have. In any case, I added the comment (http://localhost:5000/issues/667#issuecomment-306398355) At the origin of the previous PR, to see if there is not regression at the same time.

File: webcompat/static/css/development/components/markdown.css [hunk 2]
```diff
@@ -108,12 +96,29 @@
...
 
   /* pre / code */
+
+  .wc-Markdown code {
+    font-family: monospace;
+    font-size: 90%;
+    white-space: normal;
+  }
+
+  .wc-Markdown pre > code {
```
Comment (magsout) [inline webcompat/static/css/development/components/markdown.css]: @karlcow the bug is resolved by this rule and only this rule.

Maybe if you want, I can add 2 commit: "clean" and bug resolved"

**Comment by magsout** [comment]
@karlcow  I have pushed new commit, the review will be easier.

Sorry.

File: webcompat/static/css/development/components/markdown.css (hunk 1 above)
Comment (karlcow) [inline webcompat/static/css/development/components/markdown.css]: @magsout your fix is working but introducing a new issue. see the line
http://imgur.com/a/H7nw6

Comment (magsout) [inline webcompat/static/css/development/components/markdown.css]: @karlcow we have the same render see https://cl.ly/2E3I1Q1x2k1T

**Comment by karlcow** [comment]
ah ignore my comment. I probably know why. 
OK let's merge this.

Thanks @magsout

File: webcompat/static/css/development/components/markdown.css (hunk 1 above)
Comment (magsout) [inline webcompat/static/css/development/components/markdown.css]: capture d ecran 2017-06-08 a 08 13 27

**Pull request merged**